src/codeu/chat/client/simplegui2/Controller.java

```diff
@@ -0,0 +1,4 @@
+package codeu.chat.client.simplegui2;
```

Comment:
This one I wasn't sure if Javafx required a controller class to function....so I had left it in there but I will research and remove if necessary